### PR TITLE
🚒🌲🔥🧑‍🚒✈️ [WIP] US-CP Wildfire Update

### PR DESCRIPTION
**Pull request by Gooochy**
**What kind of update does this PR provide?** *Please check*

- [x] new translations / updated translations / translation fixes
- [x] a new feature
- [ ] a new module
- [ ] a bugfix for an existing feature / module
- [ ] improvement of style or user experience
- [ ] other: *Please fill out*

**Which language(s) did you add/update translations for?**
*use the xx_XX notation for exact language!*
* en_US
* ...

**What is included in your update?**
* NEW CP content

**Additional notes**
Please add any further notes here!
Heavy WIP

### CODE REVIEW

File: src/modules/extendedCallWindow/i18n/en_US.json [hunk 1]
```diff
@@ -82,7 +82,8 @@
         "foam": "gal. foam",
         "missing": "Missing on mission",
         "selected": "Selected",
-        "staff": [{ "texts": ["SWAT Personnel"], "vehicles": [16, 26] }],
+        "staff": [{ "texts": ["SWAT Personnel"], "vehicles": [16, 26] }], 
+        [{ "texts": ["Required Smoke Jumpers (in Smoke Jumper Planes)"],  "vehicles":[76] }],
```
Comment (jxn-30) [inline src/modules/extendedCallWindow/i18n/en_US.json:86]:
```suggestion
        "staff": [{ "texts": ["SWAT Personnel"], "vehicles": [16, 26] }, { "texts": ["Required Smoke Jumpers (in Smoke Jumper Planes)"],  "vehicles":[76] }],
```

**Review by jxn-30 — CHANGES_REQUESTED** [review]

File: src/modules/extendedCallWindow/i18n/en_US.json [hunk 2]
```diff
@@ -249,7 +249,11 @@
             {
                 "texts": ["Coastal Guard Plane", "Coastal Guard Planes"],
                 "vehicles": [72]
-            }
+            },
+            { 
+                "texts": ["Wildland MCC", "Wildland MCCs"], "vehicles": [74]
+            },
+                "texts":["Wildland Lead Plane", "Wildland Lead Planes", "vehicles": [75]
```
Comment (jxn-30) [inline src/modules/extendedCallWindow/i18n/en_US.json:256]:
```suggestion
                {"texts":["Wildland Lead Plane", "Wildland Lead Planes", "vehicles": [75]}
```

**Review by jxn-30 — CHANGES_REQUESTED** [review]

**Comment by Gooochy** [comment]
@jxn-30 Thank you so much for your help, this was a big challenge for me and your support is greatly appricated!

File: src/modules/missionHelper/i18n/en_US.json [hunk 3]
```diff
@@ -37,6 +37,8 @@
         "airport": "Airport Extension | Airport Extensions",
         "atf_count": "ATF Expansion | ATF Expansions",
         "brush_extension": "Forestry Expansion | Forestry Expansions",
+        "brush_command": "Wildland Commando Extension | Wildland Commando Extensions",
+        "brush_air_command": "Wildland Air Command Extension" | "Wildland Air Command Extensions",
```
Comment (jxn-30) [inline src/modules/missionHelper/i18n/en_US.json:41]:
```suggestion
        "brush_air_command": "Wildland Air Command Extension",
```

**Review by jxn-30 — CHANGES_REQUESTED** [review]

File: src/i18n/en_US.ts [hunk 4]
```diff
@@ -1073,6 +1073,76 @@ export default {
...
+        special: 'Towed by Crew cab semi',
+    },
+    78: {
+        caption: 'Tanker Trailer',
+        color: '#9f1616',
+        coins: 10,
+        credits: 5000,
+        minPersonnel: 0,
+        maxPersonnel: 0,
+        Wtank: 1000,
```
Comment (Suchty112) [inline src/i18n/en_US.ts:1142]: wtank not Wtank

**Review by Suchty112 — CHANGES_REQUESTED** [review]

**Comment by Suchty112** [comment]
what about the extensions in the buildings? ^^

File: src/i18n/en_US.ts (hunk 4 above)
Comment (Gooochy) [reply src/i18n/en_US.ts]: Is fixed

**Comment by Suchty112** [comment]
@LSS-Manager deploy

**Comment by Suchty112** [comment]
@LSS-Manager deploy

**Comment by jxn-30** [comment]
@LSS-Manager deploy

**Comment by jxn-30** [comment]
@LSS-Manager deploy

**Review by Suchty112 — APPROVED** [review]
LGTM

**Review by jxn-30 — APPROVED** [review]

**Pull request merged**